src/main/java/servlets/OAuthCallbackServlet.java

```diff
@@ -38,7 +38,7 @@
 @WebServlet("/api/oauth/callback/sheets")
 public class OAuthCallbackServlet extends HttpServlet {
   private static final Logger LOGGER = Logger.getLogger(LoginServlet.class.getName());
-  private final String SECRET_FILEPATH = "../../src/main/resources/secret.txt";
+  private final String SECRET_FILEPATH = "../../src/main/secrets/secret.txt";
```

Comment:
same here for the change in structure. I feel like this should go in a separate PR, but not confident

Reply:
It could. It was more or less a slight change in file path